src/style.css

```
@@ -1 +1,74 @@
-/* Styles go here */
+
+html {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  margin: 0;
```

```suggestion
  margin: 0 auto;
```
so it will be always at the center of the page